types/sinon-chrome/index.d.ts

```diff
@@ -353,6 +356,23 @@ declare namespace SinonChrome.permissions {
     export var request: SinonChromeStub;
 }
 
+declare namespace SinonChrome.plugins {
+    export interface Plugin {}
```

Comment:
Implementing this (in the plugins below) doesn't do anything at all, and requiring for it as the type for the `registerPlugin` param doesn't do anything beyond `plugin: {}`, which doesn't seem useful.

We could drop this, and just use `plugin: {}` above, or change this to be a type enumeration limited to these existing plugins (`export type Plugin = I18nPlugin | CookiePlugin`) or change the types here to properly represent what's required, which I think is something like `plugin: { install: (chrome: SinonChrome) => void }` (based on https://github.com/acvetkov/sinon-chrome/blob/master/src/api/index.js#L47-L57). What do you think?

Reply:
Thanks for the feedback.
I make it optional.